docs/configuration.asciidoc

```diff
@@ -1277,6 +1297,7 @@ you must instead set the `LogLevel` for the internal APM logger under the `Loggi
 | <<config-secret-token,`SecretToken`>> | No | Reporter
 | <<config-server-cert,`ServerCert`>> | No | Reporter
 | <<config-server-url,`ServerUrl`>> | No | Reporter
+| <<config-use-windows-credentials,`UseWindowsCredentials`>> | No | Reporter
 | <<config-service-name,`ServiceName`>> | No | Core
 | <<config-service-node-name, `ServiceNodeName`>> | No | Core
 | <<config-service-version,`ServiceVersion`>> | No | Core
```

Comment:
It looks like config variables are mostly alphabetized in this document. Is there a reason that this is the spot you chose? Maybe we can move this down after `UseElasticTraceparentHeader` to be more consistent?

Reply:
Originally  before z1c0 suggestion for rename it was named with Server prefix and I put it next to the server-cert and server-token properties, because I thought that it grouped nicely in that theme because it is similar setting of APM agent connection to APM server. (that was also the reason I put the server prefix for that property). So after rename I have no objection to position it wherever fits best.

Reply:
Gotcha 👍. I think I'd still vote for alphabetical, but I'll leave it up to @z1c0.

Thanks for this contribution!

Reply:
Agreed @bmorelli25, let's keep things consistent and go for alphabetical.

Reply:
moved description bellow `UseElasticTraceparentHeader `